File: ProgrammingFundamentals/C#/03.DataTypesAndVariables/Exercises/P05_BooleanVariable/P05_BooleanVariable.cs
using System;

namespace P05_BooleanVariable
{
    class P05_BooleanVariable
    {
        static void Main(string[] args)
        {
            string input = Console.ReadLine();
            bool boolean = Convert.ToBoolean(input);

            Console.WriteLine(boolean ? "Yes" : "No");
        }
    }
}

File: ProgrammingFundamentals/Java/03.DataTypesAndVariables/Exercises/src/P05_BooleanVariable.java
import java.util.Scanner;
public class P05_BooleanVariable {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        String input = console.nextLine();
        boolean convertToBool = Boolean.valueOf(input);
        System.out.println(convertToBool ? "Yes" : "No");
    }
}
